.jenkins-windows.sh

```diff
@@ -2,16 +2,16 @@
 
 export PATH=/usr/bin:/usr/local/bin:$PATH
 
-git clone https://github.com/Microsoft/vcpkg.git
-bootstrap-vcpkg.bat
-vcpkg integrate install
+git clone https://github.com/Microsoft/vcpkg.git /cygdrive/c/Program\ Files\ \(x86\)/
+/cygdrive/c/Program\ Files\ \(x86\)/bootstrap-vcpkg.bat
+/cygdrive/c/Program\ Files\ \(x86\)/vcpkg integrate install
```

Comment:
This is not writable directory, so it's not going to work even if you get all the path quoting right.

Reply:
You probably want something more like

```
git clone https://github.com/Microsoft/vcpkg.git
vcpkg/bootstrap-vcpkg.bat
vcpkg/vcpkg install giflib libjpeg-turbo libpng cairo glib tiff libexif glib pango
```